### PR TITLE
44: donuts layout

### PR DESCRIPTION
**Pull request by adamrneary**

### CODE REVIEW

File: examples/public/coffee/multi-donut.coffee [hunk 1]
```diff
@@ -1,46 +1,123 @@
...
+  color5: "#74C476"
+]
+chart = new Tactile.Chart()
+  .element($("#example_view")[0]).width(1000).height(720).data(data)
+
+chart.addSeries [
+  name: "donut-1"
+  renderer: "donut"
+  tooltip: (d) ->
+    d.label + " " + d.val
```
Comment (ksarna) [inline examples/public/coffee/multi-donut.coffee:51]: d.label doesn't work here as from within this function you can only see attributes assigned in the dataTransform. So you need to add label: d.label to the dataTransform.

**Comment by ksarna** [comment]
Transitions are great and complex. I hope you understand more then I do from them ;) Good to go! :shipit:

**Comment by adamrneary** [comment]
@Aluman GREAT WORK. I love it. I made some tweaks to the example to try a few things out, and it is very strong. Only enhancement before merging:

```
innerRadius: 70
outerRadius: 120
stackedInnerRadius: 200
stackedOuterRadius: 330
stackedIndex: 0
```

These should be something that we can optionally override, but shouldn't be required. I pull them in my last commit, and everything works right except for the layout when they are in multiples. I think it just needs to automatically assign the stackedIndex.

**Comment by adamrneary** [comment]
@Aluman also, there's a minor bug:
- click "stack" and then click "stack" again
- click "unstack" and then click "unstack" again

If it's already in that state, it shouldn't explode like that.

**Comment by Aluman** [comment]
Default properties:
    stackedInnerRadius: 200
    stackedOuterRadius: 330
    innerRadius: 70
    outerRadius: 120

Default Indexes: 0 for zero series, 1 for first series etc.

**Comment by adamrneary** [comment]
AWESOME.

**Pull request merged**